Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -10,58 +10,66 @@ import UIKit
 class ReminderDetailEditDataSource: NSObject {
     
     enum ReminderSection: Int, CaseIterable {
+        
+        // MARK: Cases
```

Comment:
Do not put MARK everywhere. They are not like comments. Use only where they are necessary.

Reply:
The reason i put MARK here was cause the enum was quite large cause. There were some methods and computed properties too along with cases. Nevertheless, will remove it now.